manifests/j/JetBrains/dotUltimate/2022.1/JetBrains.dotUltimate.installer.yaml

```
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+PackageIdentifier: JetBrains.dotUltimate
+PackageVersion: "2022.1"
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallerSwitches:
+  Silent: /Silent=True
+  SilentWithProgress: /Silent=True
+  Custom: /VsVersion=16.0 /SpecificProductsNames=ReSharper;dotTrace;dotMemory;dotCover;dotPeek;Rider;ReSharperCpp;teamCityAddin
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.jetbrains.com/resharper/dotUltimate.2022.1/JetBrains.ReSharper.2022.1.web.exe
```

```suggestion
  InstallerUrl: https://download.jetbrains.com/resharper/dotUltimate.2022.1/JetBrains.ReSharper.2022.1.web.exe
```
Should this be `https://download-cdn.jetbrains.com/resharper/dotUltimate.2022.1/JetBrains.dotUltimate.2022.1.exe` or was the URL changed?